src/App.test.js

```diff
@@ -1,5 +1,5 @@
 import { render, screen } from '@testing-library/react';
-import App from './App';
+import App from './components/App';
 
```

Comment:
- [ ] Kindly delete this file. Remember that part of the project requirements is for you to delete all _**boilerplates**_ created with _**Create React App (CRA)**_